setup.py

```diff
@@ -42,4 +42,13 @@
         ],
     },
     include_package_data=True,
+    classifiers=[
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.4",
```

Comment:
Python 3.4 is already EOL.  Perhaps add 3.5 and 3.7 instead?

https://devguide.python.org/devcycle/#end-of-life-branches

Reply:
Yeah, I know but didn't want to diverge from the setup in Travis. Dropping a Python version seems like a thing better suited for a different PR.